### ISSUE TITLE
Problem in Divider

### ISSUE THREAD

**Issue by manoharmohanta** [danshanley/FPU#1]
I have tested the code by giving Same values in a and b and the output not equal to 1.
![capture](https://user-images.githubusercontent.com/6312750/33471940-b62f5592-d695-11e7-94f8-aec9f666da83.JPG)

**Comment by Silent-Zebra** [comment]
Agreed, division doesn't work unfortunately. 0.6/0.8 gives 11.7 as the result.

A bit of a shame because I really liked the way everything else was set out and the ease of use.
